hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/container/balancer/TestContainerBalancerDatanodeNodeLimit.java

```diff
@@ -79,6 +96,7 @@ public void containerBalancerShouldObeyMaxDatanodesToInvolveLimit(@Nonnull Mocke
     }
     config.setIterations(1);
     config.setMaxSizeToMovePerIteration(50 * STORAGE_UNIT);
+    mockedSCM.init(config);
```

Comment:
Relating to the above comment, we would not require to call init here specifically. Similarly for all the test cases in this class.
What is the reason to remove init call within startBalancerTask?

Reply:
Thanks, LGTM